hibernate-core/src/test/java/org/hibernate/orm/test/type/SQLServerNVarCharTypeTest.java

```diff
@@ -33,7 +32,7 @@
  */
 @TestForIssue(jiraKey = "HHH-10529")
 @RequiresDialect(value = SQLServerDialect.class, majorVersion = 10)
```

Comment:
Since version 10 is for SQL Server 2008, you can now drop the major version 10.

```suggestion
@RequiresDialect(value = SQLServerDialect.class)
```